soc/common/riscv-opensbi/sbi_ecall_interface.h

```diff
@@ -0,0 +1,436 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Zephyr doesn't accept non-Apache 2.0 licensed files into its tree by default. This is something that would need to be approved by the Governing Board. See more here: https://docs.zephyrproject.org/latest/contribute/guidelines.html#components-using-other-licenses

Reply:
just wondering, if I am not completely wrong, then Apache 2.0 should include BSD-2. So wouldnt it be possible to replace legally with Apache 2.0?